src/libs/SelectionScraper/index.js

```diff
@@ -0,0 +1,115 @@
+import render from 'dom-serializer';
+import ExpensiMark from 'expensify-common/lib/ExpensiMark';
+import {parseDocument} from 'htmlparser2';
+import _ from 'underscore';
+
+const elementsWillBeSkipped = ['html', 'body'];
+const tagAttribute = 'data-testid';
+
+/**
+ * Reads html of selection. If there is no Selection API returns empty string.
```

Comment:
If there is no selection, the API

Reply:
I meant if browser doesn't support [Selection API](https://caniuse.com/selection-api), Is it OK if I update it like that?

Reply:
Hi @thienlnam, I've updated all comments, can you recheck, if they are OK?

Reply:
Yes that update is good 👍  (just missing one more comment I left unresolved)